app/helpers/test_executions_results_helper.rb

```diff
@@ -23,7 +23,7 @@ def supplemental_data_errors(errors)
       err.has_attribute?('error_details') && err['error_details'].key?('type') && err['error_details']['type'] == 'supplemental_data'
     end
     # sort by population key (IPP, DENOM, ...)
-    return_errors.sort do |a, b|
+    return_errors.sort do |a, _b|
```

Comment:
I didn't touch that file so it must be something that rubocop put in there

On Tue, Mar 8, 2016 at 1:24 PM, dehall <notifications@github.com> wrote:

> In app/helpers/test_executions_results_helper.rb
> <https://github.com/projectcypress/cypress/pull/387#discussion_r55402726>:
>
> > @@ -23,7 +23,7 @@ def supplemental_data_errors(errors)
> >        err.has_attribute?('error_details') && err['error_details'].key?('type') && err['error_details']['type'] == 'supplemental_data'
> >      end
> >      # sort by population key (IPP, DENOM, ...)
> > -    return_errors.sort do |a, b|
> > +    return_errors.sort do |a, _b|
>
> i think the fix here should be to replace the duplicate
> a.error_details.population_key with b.error... instead of setting b to
> ignored. this is a separate issue and unrelated to the core change though
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projectcypress/cypress/pull/387/files#r55402726>.
>